examples/java/src/main/java/org/apache/ignite/example/storage/PersistentPageMemoryStorageExample.java

```diff
@@ -30,6 +30,6 @@ public class PersistentPageMemoryStorageExample {
      * @throws Exception If failed.
      */
     public static void main(String[] args) throws Exception {
-        new StorageEngineExample("persistent").run();
+        new StorageEngineExample("default").run();
```

Comment:
The [README.md](https://github.com/apache/ignite-3/blob/main/examples/java/README.md) states that before running an example one should start an ignite 3 node with the config provided with examples. That config has the `persistent` profile configured. When a node is started this way the `RocksDbStorageExample` and `VolatilePageMemoryStorageExample` also works as expected. 

Hence there is no need to change this line.